tests/core/ingestion/test_physical_document.py

```diff
@@ -14,7 +14,7 @@
 def test_physical_document_from_row(test_db: Session):
     populate_for_ingest(test_db)
     row = DocumentIngestRow.from_row(1, get_doc_ingest_row_data(0))
-    row.language = "English"
+    row.language = ["English", "German"]
```

Comment:
We probably should test for `from_row` too (`tests/core/ingestion/test_ingest_row.py`)

i.e. change 0 index to:
```
1001,0,Test1,FALSE,FALSE,N/A,Collection1,CollectionSummary1,Title1,Fam1,Summary1,,MAIN,,DZA,http://place1|en,executive,02/02/2014|Law passed,Energy,,,Mitigation,,Order,,,English;German,Energy Supply,Algeria,,,CCLW.executive.1001.0,CCLW.family.1001.0,CPR.Collection.1,FamSlug1,DocSlug1
```